locale/en/blog/weekly-updates/weekly-update.2015-11-06.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Weekly Update - Nov 6th, 2015
+author: Minwoo Jung (@jmwsoft)
+date: 2015-11-06T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2015-11-06
+layout: blog-post.hbs
+---
+
+### Node.js News — November 6th
+Node.js v4.2.2 (LTS) is released
+
+### Node.js v4.2.2 (LTS) Releases
+
+This week we have one release: [Node.js v4.2.2 (LTS)](https://nodejs.org/en/blog/release/v4.2.2/). Complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
+
+### Notable changes
+
+This is an LTS maintenance release that addresses a number of issues:
+
+* [[`1d0f2cbf87`](https://github.com/nodejs/node/commit/1d0f2cbf87)] - **buffer**: fix value check for writeUInt{B,L}E (Trevor Norris) [#3500](https://github.com/nodejs/node/pull/3500)
+* [[`2a45b72b4a`](https://github.com/nodejs/node/commit/2a45b72b4a)] - **buffer**: don't CHECK on zero-sized realloc (Ben Noordhuis) [#3499](https://github.com/nodejs/node/pull/3499)
+
+### Node.js foundation is considering an iteration on the official logo
+
+* [Please click here](https://github.com/nodejs/evangelism/issues/179) to post images and ideas. 
```

Comment:
Maybe instead of `Please click here`, we can go with something more like:

    * [Submit your images and ideas on GitHub!](https://github.com/nodejs/evangelism/issues/179)